app/javascript/react/screens/App/Overview/OverviewValidators.js

```diff
@@ -0,0 +1,222 @@
+import { array, string, object } from 'yup';
+import { logSchemaError } from '../../../../common/schemaHelpers';
+
+const providersSchema = array()
+  .of(
+    object().shape({
+      href: string().required(),
+      type: string().required(),
+      id: string().required()
+    })
+  )
+  .nullable();
+
+export const validateProviders = providers => {
```

Comment:
Looks like we do this `validate` function a few times and in large its the same each time, would be nice to abstract this from each schema to have one generic validation tool to export

Reply:
this can be done just calling validate on a passed in schema in a generic method, will update.